builtin/providers/aws/resource_aws_elb.go

```diff
@@ -213,38 +231,16 @@ func resourceAwsElbCreate(d *schema.ResourceData, meta interface{}) error {
 
 	d.Set("tags", tagsToMapELB(tags))
 
-	if d.HasChange("health_check") {
```

Comment:
I removed this because create calls update and update calls this exact code again.